email-client/src/main/java/uk/gov/hmcts/cmc/email/AppInsightsService.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.cmc.email;
+
+import com.microsoft.applicationinsights.TelemetryClient;
+
+import static java.util.Collections.singletonMap;
+
+public class AppInsightsService {
```

Comment:
Do we really need this wrapping service? How about using the `TelemetryClient` instead?

Reply:
inlined